core/SCsub

```diff
@@ -194,10 +194,11 @@ SConscript("string/SCsub")
 SConscript("config/SCsub")
 SConscript("error/SCsub")
 
+core_env = env.Clone()
 
 # Build it all as a library
-lib = env.add_library("core", env.core_sources)
+lib = core_env.add_library("core", env.core_sources)
 env.Prepend(LIBS=[lib])
 
 # Needed to force rebuilding the core files when the thirdparty code is updated.
-env.Depends(lib, thirdparty_obj)
+core_env.Depends(lib, thirdparty_obj)
```

Comment:
Depends is primarily acting just on the node, so the calling environment doesn't usually matter, unless your using special substitution or overwriting the Depends method. This should probably just be `env.Depends` to stay consistent.

Reply:
fixed this to be consistent again